.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,35 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.192.0/containers/dotnet
+{
+	"name": "C# (.NET)",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"args": { 
+			// Update 'VARIANT' to pick a .NET Core version: 2.1, 3.1, 5.0
+			"VARIANT": "5.0",
+		}
+	},
+
+	"settings": {
+		// Loading projects on demand is better for larger codebases
+		"omnisharp.enableMsBuildLoadProjectsOnDemand": true
+	},
+
+	// Add the IDs of extensions you want installed when the container is created.
+	"extensions": [
+		"ms-dotnettools.csharp"
+	],
+
+	// Use 'postCreateCommand' to run commands after the container is created.
+	"postCreateCommand": "bash -i ${containerWorkspaceFolder}/.devcontainer/scripts/postCreateCommand.sh",
+
```

Comment:
Would a "postStartCommand" allow us to do something useful? If we could write a welcome message to the user's terminal, we could help get them started by displaying some common build commands, a link to a doc, etc.

Reply:
Maybe we could link to a doc and also run `./build.sh --help`?

Reply:
What build commands or docs do you think are useful here?

Looking around, I don't see others doing this. Maybe it is something we can enable after the initial work?

Reply:
>  also run ./build.sh --help?

That output is way too verbose to be useful for a new contributor.

If anything here, I could imagine pointing to https://github.com/dotnet/runtime/blob/main/docs/workflow/README.md. Beyond that, I'm not sure of the value of dumping more things out.

Reply:
Yeah, I only found a few examples: https://grep.app/search?current=3&q=postStartCommand&filter[path][0]=.devcontainer/ and didn't spot any that were simply dumping help.

I had in mind just writing a few lines welcoming folks and pointing them to the readme, and saying to type "./build.sh --help"

yes, no need to do it now unless you feel like it. Just trying to think about how to smooth the path after the dev container starts and they're wondering what to do next. that is, if it even is visible in the terminal. I'm guessing it is.